src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (newRobot[value]) {
```

Comment:
Change the condition to check if the property exists in newRobot. Use `newRobot.hasOwnProperty(value)` instead of 'if (newRobot[value])'.